lib/macho/macho_file.rb

```diff
@@ -250,14 +250,13 @@ def rpaths
       command(:LC_RPATH).map(&:path).map(&:to_s)
     end
 
-    # Changes the runtime path `old_path` to `new_path`
+    # Changes the relative path `old_path` to `new_path`
```

Comment:
The “R” definitely doesn't stand for “relative”. `dyld(1)` talks about a “run path list” that is populated from the various `LC_RPATH` load commands, thus I'd make this “run path” or leave it as “runtime path”.